src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let stateCopy = { ...state };
+
+  actions.forEach((action) => {
+    switch (action.type) {
```

Comment:
You can use destruction for `action`
```suggestion
  actions.forEach((action) => {
    const { type, ...... } = action;
  
    switch (type) {
```